eoscostarica/eosio-spend-explorer
# feat(webapp): improve UI from main page

### Title

### What does this PR do?

- Resolve #53 

### Steps to test

1. run the project and open localhost:3000 in your browser
2. click on income in the sidebar and see new features

#### CheckList
- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check
